edb/server/pgcon/pgcon.pyx

```diff
@@ -949,6 +981,22 @@ cdef class PGProto:
         buf.write_bytestring(stmt_name)
         return buf.end_message()
 
+    cdef make_auth_password_md5_message(self, bytes salt):
+        cdef WriteBuffer msg
+
+        msg = WriteBuffer.new_message(b'p')
+
+        user = self.pgaddr.get('user') or ''
+        password = self.pgaddr.get('password') or ''
+
+        # 'md5' + md5(md5(password + username) + salt))
```

Comment:
Is this encoding of a password is still considered safe? I would expect PBKDF with thousands of iterations here too. (perhaps, argon2_ffi?)

Reply:
This is what RDS Postgres advertises.  Ideally, they'd need to configure it to use SCRAM, but they didn't.

Reply:
Ah, sorry, I've misread this as our method for storing passwords.